src/statemachine/blender.js

```diff
@@ -0,0 +1,250 @@
+pc.extend(pc, (function () {
```

Comment:
BTW as a naming convention for the files we folder_filename. So this file for example should be named statemachine_blender.js